package.json

```diff
@@ -42,10 +43,10 @@
   },
   "devDependencies": {
     "cpy-cli": "^5.0.0",
-    "hugo-extended": "0.126.1",
-    "markdown-link-check": "^3.12.1",
+    "hugo-extended": "0.126.3",
```

Comment:
Very nice that `npm-check-updates -u` respects the pinned nature of `hugo-extended`!